kong.conf.default

```diff
@@ -371,6 +371,7 @@
 #pg_user = kong                  # The username to authenticate if required.
 #pg_password =                   # The password to authenticate if required.
 #pg_database = kong              # The database name to connect to.
+#pg_schema =                     # The database schema to be used.
```

Comment:
Let's clarify what happens when this value is empty (as with the default) in the comment. Something like:

> If unspecified, Kong will respect the `search_path` value of your PostgreSQL instance.

Or something simple like this.

Reply:
Added.